CL/cl_ext.h

```diff
@@ -2446,6 +2446,18 @@ typedef struct _cl_queue_family_properties_intel {
 ***************************************************************/
 #define cl_intel_sharing_format_query 1
 
+/***************************************************************
+* cl_ext_image_from_buffer
+***************************************************************/
+
+#ifdef CL_VERSION_3_0
```

Comment:
Strictly speaking, there is nothing here that requires OpenCL 3.0.

I do see from the extension spec that it requires OpenCL 3.0, though I think that's more because it requires support for cl_ext_image_requirements_info, which requires OpenCL 3.0?

Reply:
Yes exactly, this extension depends on cl_ext_image_requirements_info which requires OpenCL 3.0